posts/2020/03/Spyder-4.md

```diff
@@ -0,0 +1,31 @@
+
+I also want to thank our community for its continued support. As I've witnessed during my years as Spyder's lead developer, many newbies and veterans alike keep choosing Spyder as a viable alternative to do scientific programming in Python. It's really you, the members of this fantastic community, which maintain Spyder relevant in a highly competed field by using and contributing back to it. I'm honored and humbled to have received such first-class additions to Spyder 4 from our volunteer contributors as our new Plots pane, thanks to [Jean-Sébastien Gosselin](https://github.com/jnsebgosselin); a complete re-architecturing of our IPython console, which allowed a lot of improvements to our debugger, thanks to [Quentin Peter](https://github.com/impact27); auto-saving functionality for our editor, thanks to [Jitse Niesen](https://github.com/jitseniesen); the initial implementation of the `runcell` command, thanks to [Brian Olsen](https://github.com/bcolsen); and significant improvements to our documentation, thanks to [CAM Gerlach](https://github.com/CAM-Gerlach).
+
+Besides, when the future looked grim, after my team and I were let go from Anaconda at the end of 2017 (not by Travis decision), a lot of users came to our rescue by making donations through our [Open Collective page](https://opencollective.com/spyder). That, and a Numfocus development grant we received next year, filled us with confidence and allowed us to continue with Spyder's development in 2018, even after losing part of our team in the process.
+
+Last year it was also a pleasant surprise to learn that several companies were interested in seeing Spyder prosper and thrive. Through Quansight Labs and its [Community Working Order](https://labs.quansight.org/blog/2019/05/community-driven-opensource-funded-development/) concept, we were able to sign contracts with two of them: [TDK-Micronas](https://www.micronas.tdk.com/en) and [Kite](https://kite.com/). Their support was critical to finish Spyder 4 because it allowed me to hire most of my old Anaconda team back, plus two new additions, to work full-time on the project. Therefore, I can't them enough for showing up just at the right time!
```

Comment:
```suggestion
Last year it was also a pleasant surprise to learn that several companies were interested in seeing Spyder prosper and thrive. Through Quansight Labs and its [Community Work Order](https://labs.quansight.org/blog/2019/05/community-driven-opensource-funded-development/) concept, we were able to sign contracts with two of them: [TDK-Micronas](https://www.micronas.tdk.com/en) and [Kite](https://kite.com/). Their support was critical to finish Spyder 4 because it allowed me to hire most of my old Anaconda team back, plus two new additions, to work full-time on the project. Therefore, I can't them enough for showing up just at the right time!
```

To be fair, I'm not saying you should change it but realistically, I highly doubt Kite gives a shit about Spyder itself, they just see at as a marketing platform to agglomerate more users, just like they did with the different open-source projects they bought out in the past (although, to be clear, their present behavior is vastly less objectionable than what it used to be, I don't think you would have stood for the kind of nonsense they used to pull). Assuming that funding went to substantively more than just integrating Kite, as I assume it did, I don't see it as a net negative for Spyder but at least between you and I, we can recognize it for what it is.